src/controllers/itemdetailpage.js

```diff
@@ -336,7 +336,7 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "userSetti
 
         return html = html.join(" / ");
     }
-
+    // TODO: Clean this and the CSS rules for mobile layout. See #1022
```

Comment:
```suggestion
    // TODO: Clean this and the CSS rules for mobile layout. See GH #1022
```
IIRC github has dpecial handling for such comments

Reply:
How?

Reply:
I'll remove this completely as in my next commit I'll handle this with layoutManager, so no hacky stuff involved anymore.